react/SummaryWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { useOrderForm } from 'vtex.order-manager/OrderForm'
+import { useOrderCoupon } from 'vtex.order-coupon/OrderCoupon'
+
+const SummaryWrapper: React.FC = () => {
+  const {
+    orderForm: {
+      totalizers,
+      value,
+      marketingData: { coupon },
+    },
+  } = useOrderForm()
+
+  const { insertCoupon } = useOrderCoupon()
+
+  return (
+    <div className="mh8-m mh0-l pt6 pt0-l bt b--muted-4 bn-l">
+      <ExtensionPoint
+        id="checkout-summary"
+        // @ts-ignore
```

Comment:
Could you add to the comment an explanation as to why this is necessary? Something like `// @ts-ignore: <name of the TS error>`

Reply:
sure! TIL

Reply:
actually, using the prop `blockProps` enables me to remove this ts-ignore :-)

Reply:
Even better 👍